package-rfcx/rfcx/client.py

```diff
@@ -228,3 +229,94 @@ def downloadGuardianAudio(self, dest_path=None, guardian_id=None, min_date=None,
 
         return audio.downloadGuardianAudio(self.credentials.id_token, dest_path, guardian_id, min_date, max_date, file_ext, parallel)
 
+    # v2
+    def saveAudioFile(self, dest, audio_id, file_ext='opus'):
+        """ Prepare `url` and `local_path` and save it using function `__save_file` 
+        Args:
+            dest_path: Audio save path.
+            audio_id: RFCx audio id in format `{stream_id}_t{start time}.{end time}_g{audio gain}_f{audio format}`
```

Comment:
There are some important points here:

1. We don't want the user of the SDK have to understand the naming of the files. They should be able to specify: stream, start, end, format.
2. You can download any start/end, which the user might want to do... they can download each detection as an audio file (so you wouldn't need to cut the audio).